dags/pull_miovision.py

```diff
@@ -70,7 +78,51 @@ def check_partitions():
         retries = 0,
         trigger_rule='none_failed'
     )
+    
+    @task_group()
+    def data_checks():
+        """Task group to check critical data quality measures which could warrant re-running the DAG."""
+        data_check_params = {
+            "table": "miovision_api.volumes_15min_mvt",
+            "lookback": '60 days',
+            "dt_col": 'datetime_bin',
+            "threshold": 0.7
+        }
+        check_row_count = SQLCheckOperatorWithReturnValue(
+            task_id="check_row_count",
+            sql="select-row_count_lookback.sql",
+            conn_id="miovision_api_bot",
+            retries=1,
```

Comment:
I don't think we need retries for tasks in data checks group.

Reply:
Fixed!